src/components/ArticleHeader/ArticleHeader.module.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../scss/bootstrap.scss";
+
+.articleHeader {
+  background-image: linear-gradient(top, rgba(0,0,0,0.5), rgba(0,0,0,0.5)),url("../../img/banner.jpg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 5em 3em 3em 3em;
```

Comment:
I removed a `marging top: 3em` here and added it to the padding. So the bg image stays the same height.